pkg/apiserver/resthandler.go

```diff
@@ -470,11 +470,11 @@ func PatchResource(r rest.Patcher, scope RequestScope, typer runtime.ObjectTyper
 		ctx := scope.ContextFunc(req)
 		ctx = api.WithNamespace(ctx, namespace)
 
-		// PATCH requires same permission as UPDATE
-		if admit.Handles(admission.Update) {
+		if admit != nil && admit.Handles(admission.Patch) {
 			userInfo, _ := api.UserFrom(ctx)
 
-			err = admit.Admit(admission.NewAttributesRecord(obj, scope.Kind.GroupKind(), namespace, name, scope.Resource.GroupResource(), scope.Subresource, admission.Update, userInfo))
+			// TODO this record should include the patch itself.  Right now we can't do that because patches are not runtime.Objects
```

Comment:
The comment line or the code? To be honest, I think that adding `Patch   Operation = "PATCH"` and using it here is just the wrong direction to go entirely. Removing it would make me go away and leave you alone :) :)

Even if you want to leave refactoring update to take old & new to a later date--which sounds fine--adding the patch admission in and leaving it like this seems like it's adding additional broken things.

Does something super bad happen if you make the changes in line 506 but not 477?